apps/cookbook/src/app/examples/modal-example/first-embedded-modal-example/first-embedded-modal-example.component.html

```diff
@@ -54,6 +61,7 @@ <h3>Open nested:</h3>
       </button>
       <cookbook-example-configuration-wrapper>
         <cookbook-modal-example-configuration
+          [(showPageProgress)]="showPageProgress"
```

Comment:
Add separate prop for that, otherwise it toggles the page progress for the current view as well
```suggestion
          [(showPageProgress)]="showNestedPageProgress"
```